src/components/banners/Banner.tsx

```diff
@@ -7,6 +7,9 @@ import CheckCircleIcon from '@material-ui/icons/CheckCircle';
 import InfoIcon from '@material-ui/icons/Info';
 import NotificationsIcon from '@material-ui/icons/Notifications';
 import CloseIcon from '@material-ui/icons/Close';
+// Collapsible icons
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
```

Comment:
CoreUI has some native arrow up and down icon components. What do you think about using those instead of pulling from MUI?

Reply:
@chowington Thank you for your comment. I didn't know there were native icon but found that the closest ones are CaretUp/CaretDown. Two things I noticed: a) default color of CaretUp/CaretDown is gray but I think it can be adjusted with color so no issue here; b) this one is arguable, but I like MUI's mouseover event which will help user find out the action more easily: though most may already know that 😃 . But the CaretUp/CaretDown does not support such an event. 

I will leave the current one as it is but added the possibility of using CaretUp/CaretDown by commenting it.

Reply:
Ok! I do believe you could emulate the behavior either by

1. Using the `css` prop, which (I think?) supports the `:hover` effect
2. Implementing it manually using something like `[isHoveredOn, setIsHoveredOn] = useState()` and the native `mouseenter/mouseleave` events (I guess you'd have to use them on a wrapper component around the icon)

Details are rough, sorry lol. I'll approve anyway if you decide not to go those routes!

Reply:
@chowington Perhaps those would work (👍) but I am hesitant to do that now as it adds more works that are not that urgent, I suppose. Thank you for your detailed reviews! 👍